scripts/update-docker.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Used to restart a single msgr api container running on a server
+# First kills all running containers named msgr_api then starts a new container with the name msgr_api
+
+msgr_containers=$(docker ps -q --filter name="msgr_api")
+
+if [ ! -z $msgr_containers ] 
+then
+    echo "Killing all running docker processes"
+    docker kill $msgr_containers
```

Comment:
Could you add a remove step after the kill step? Otherwise, the script will fail since the container is killed but not removed

Reply:
:+1: this pr is giving me doubts about every bash script I've ever written